config/rwahs/components/providence

```diff
@@ -27,7 +28,10 @@ function postDeploy {
     localSettingsRoot="${2}"
 
     ln -s "${localSettingsRoot}/setup.php" "${targetRoot}/${targetParent}/current/setup.php"
+    ln -s "${localSettingsRoot}/media/collectiveaccess" "${targetRoot}/${targetParent}/current/media/collectiveaccess"
+
     "${COLLECTIVEACCESS_HOME}/support/bin/caUtils" update-database-schema
```

Comment:
I just remembered that this won't work as this waits for human input, and doesn't have a -y flag